minio/helpers.py

```diff
@@ -137,20 +135,16 @@ def parts_manager(data, part_size=5*1024*1024):
     :param part_size: Individual part number defaults to 5MB.
     :return: Returns :class:`PartMetadata <PartMetadata>`
     """
-    tmpdata = io.BytesIO()
-    md5hasher = Hasher.md5()
-    sha256hasher = Hasher.sha256()
-    total_read = 0
-    while total_read < part_size:
+    part_data = io.BytesIO()
```

Comment:
> i don't think so.. i was able to run it fine. @vadmeste

It is weird.. we can't fill 5mb with 1024*3 sized chunk since (5*1024*1024) % (1024*3) != 0